a10_neutron_lbaas/v2/handler_listener.py

```diff
@@ -88,17 +87,19 @@ def _set(self, set_method, c, context, listener):
                 # If the binding is being deleted and the port isn't https
                 # remove the port and re-create it.
```

Comment:
This comment should be "if the binding is being deleted and the listener wasn't created as https, remove the https port to make room for the original port.